hooks/useZoraMintComment.ts

```
@@ -70,24 +69,24 @@ const useZoraMintComment = () => {
 
       let hash: Address | null = null;
 
-      if (sale.pricePerToken === BigInt(0)) {
+      if (saleConfig.pricePerToken === BigInt(0)) {
         hash = await mintOnSmartWallet({
```

ESLint

[error] 'hash' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)